src/lib/modules/executor/executor.go

```diff
@@ -0,0 +1,105 @@
+package executor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/bench-routes/bench-routes/tsdb/file"
+	"github.com/go-ping/ping"
+)
+
+type Ping struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Jitter struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Monitor struct {
+	ResDelay  time.Duration `json:"resDelay"`
+	ResLength int64         `json:"resLength"`
+}
```

Comment:
```suggestion
// Response contains evaluations of response delay and length, calculated during monitoring of an API.
type Response struct {
	Delay  time.Duration `json:"delay"`
	Length int64         `json:"length"`
	Size     []byte        `json:"size"`
}
```
Let's add another field as `size` which will be in bytes. These will be raw `len()` of bytes that you get in response.